packages/office-ui-fabric-react/src/Utilities.ts

```diff
@@ -1 +1,2 @@
 export * from '@uifabric/utilities/lib/index';
+export * from './utilities/selectableOption';
```

Comment:
Unless we move the private utilities in oufr to the actual utilities library, which may not be unreasonable, I would suggest we just export with the top level imports that need it.